test/primitives-test.js

```diff
@@ -1012,104 +1026,104 @@ describe('Minim Primitives', () => {
       });
     });
 
-    describe('#filter', () => {
-      it('allows for filtering on keys', () => {
-        const foo = objectType.filter((value, key, member) => {
+    describe('#filter', function() {
+      it('allows for filtering on keys', function() {
+        var foo = objectType.filter(function(value, key, member) {
           return key.equals('foo');
         });
         expect(foo.keys()).to.deep.equal(['foo']);
       });
 
-      it('allows for filtering on values', () => {
-        const foo = objectType.filter((value, key, member) => {
+      it('allows for filtering on values', function() {
+        var foo = objectType.filter(function(value, key, member) {
           return value.equals('bar');
         });
         expect(foo.keys()).to.deep.equal(['foo']);
       });
 
-      it('allows for filtering on members', () => {
-        const foo = objectType.filter((value, key, member) => {
+      it('allows for filtering on members', function() {
+        var foo = objectType.filter(function(value, key, member) {
           return member.value.equals('bar');
         });
         expect(foo.keys()).to.deep.equal(['foo']);
       });
     });
 
-    describe('#forEach', () => {
-      it('provides the keys', () => {
-        let keys = [];
-        objectType.forEach((value, key, member) => {
+    describe('#forEach', function() {
+      it('provides the keys', function() {
+        var keys = [];
+        objectType.forEach(function(value, key, member) {
           return keys.push(key.toValue());
         });
         expect(keys).to.deep.equal(['foo', 'z']);
       });
 
-      it('provides the values', () => {
-        let values = [];
-        objectType.forEach((value, key, member) => {
+      it('provides the values', function() {
+        var values = [];
+        objectType.forEach(function(value, key, member) {
           return values.push(value.toValue());
         });
         expect(values).to.deep.equal(['bar', 1]);
       });
 
-      it('provides the members', () => {
-        let keys = [];
-        objectType.forEach((value, key, member) => {
+      it('provides the members', function() {
+        var keys = [];
+        objectType.forEach(function(value, key, member) {
           return keys.push(member.key.toValue());
         });
         expect(keys).to.deep.equal(['foo', 'z']);
       });
     });
 
-    describe('#[Symbol.iterator]', () => {
-      it('can be used in a for ... of loop', () => {
-        const items = [];
-        for (let item of objectType) {
-          items.push(item);
-        }
-
-        expect(items).to.have.length(2);
-      });
-    });
+    // describe('#[Symbol.iterator]', function() {
+    //   it('can be used in a for ... of loop', function() {
+    //     var items = [];
+    //     for (let item of objectType) {
+    //       items.push(item);
+    //     }
+    //
+    //     expect(items).to.have.length(2);
+    //   });
+    // });
```

Comment:
I wonder if it makes sense to include an ES6 test (or a few) to ensure compatibility? We could either try to detect e.g. iojs 2+ or use babel (but only on those specific tests). Thoughts?